region/brinstar/kraid/Kraid Eye Door Room.json

```
@@ -651,6 +740,40 @@
       "bypassesDoorShell": true,
       "flashSuitChecked": true
     },
+    {
+      "link": [2, 1],
+      "name": "Come in Shinecharging, Gain Flash Suit (Spikesuit) - Leave with Shinespark",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 3,
+          "openEnd": 0
+        }
+      },
+      "requires": [
+        "canFlashSuitDoubleSpark",
+        {"shineChargeFrames": 175},
+        "canDownBack",
+        "canShinechargeMovementTricky",
+        {"thornHits": 1},
+        "canSpikeSuit",
+        {"shinespark": {"frames": 20}}
```

```suggestion
        {"shinespark": {"frames": 19}}
```
It's ok to put this as 19 the same as the others (even though in my one video I sparked from further away, using an extra energy).